### PR TITLE
[velux] Updated Discovery instructions for scenes and actuators

### PR DESCRIPTION
**Pull request by jirikraus**
# Velux Binding Updated Discovery instructions for scenes and actuators

# Description

Updated Discovery instructions as after KLF200 is auto discovered and configured discovery of scenes and actuators need to be manually triggered.

# Testing

This is a documentation update.

### CODE REVIEW

**Comment by jirikraus** [comment]
I am using openHABian 3.3.0 and for Auto Discover of my KLF 200 worked as described. However the discovery of the configured actuators and scenes needed to be manually triggered so I am proposing this documentation update. I got unblocked by this https://community.openhab.org/t/velux-klf-200-binding-not-adding-things-to-inbox/120858 forum thread but thought it might be better to directly address this in the documentation.

File: bundles/org.openhab.binding.velux/README.md [hunk 1]
```diff
@@ -49,7 +49,7 @@ The binding supports the following types of Thing.
 To simplify the initial provisioning, the binding provides one thing which can be found by autodiscovery.
 The binding will automatically discover Velux Bridges within the local network, and place them in the Inbox.
 Once a Velux Bridge has been discovered, you will need to enter the `password` Configuration Parameter (see below) before the binding can communicate with it.
-And once the Velux Bridge is fully configured, the binding will automatically discover all its respective scenes and actuators (like windows and rollershutters), and place them in the Inbox.
+And once the Velux Bridge is fully configured, you need to trigger discovering its respective scenes and actuators (like windows and rollershutters). For this log into the OpenHAB webfront go to Settings -> Things and click on the + symbol in the lower right. Then select the Velux Binding and click Scan. After the scan has completed the scences and actuators configured in the KLF 200 are placed in the Inbox.
```
Comment (andrewfg) [inline bundles/org.openhab.binding.velux/README.md:52]: Each sentence must be on a new line.

**Review by andrewfg — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.velux/README.md (hunk 1 above)
Comment (jirikraus) [reply bundles/org.openhab.binding.velux/README.md]: Thanks for the review. I have applied the formatting change and also extended the documentation to also cover cases where auto discovery works and to make it robust for possible future changes enabling auto discovery. Resolving this conversation.

**Review by andrewfg — APPROVED** [review]
LGTM

**Review by lolodomo — APPROVED** [review]
OK even if it looks to me like the standard way to discover things in openHAB, not something specific to this binding.

**Pull request merged**